sound/soc/sof/sof-priv.h

```diff
@@ -492,6 +492,10 @@ void snd_sof_free_topology(struct snd_sof_dev *sdev);
 int snd_sof_complete_pipeline(struct snd_sof_dev *sdev,
 			      struct snd_sof_widget *swidget);
 
+int sof_load_pipeline_ipc(struct snd_sof_dev *sdev,
+			  struct sof_ipc_pipe_new *pipeline,
+			  struct sof_ipc_comp_reply *r);
+
```

Comment:
same a before, you are defining a function after you've used it, so git bisect is broken again.